shishito/runtime/environment/shishito.py

```diff
@@ -138,36 +139,28 @@ def add_cmdline_arguments_to_browser(self, browser_capabilities, config_section)
             except:
                 pass
 
-    def add_download_path_to_browser(self, browser_capabilities, config_section):
-        """
-        add default path to downloading files from qa-bstack
-        :param browser_capabilities:
-        :param config_section:
-        :return:
-        """
+    def set_download_path(self, driver):
+        """set download path for Chrome. Variable in config file download_path can be either string or env var."""
         try:
-            browser_name = self.shishito_support.get_opt(config_section, 'browser').lower()
-            download_file_path = self.shishito_support.get_opt('download_path')
-            m = re.match('^\$([A-Z][A-Z_]+)$', download_file_path)
-
+            download_directory = self.shishito_support.get_opt('download_path')
+            m = re.match('^\$([A-Z][A-Z_]+)$', download_directory)
             if m is not None:
```

Comment:
I am not sure about this whole `if` section. What is it trying to solve?

Reply:
I get it that the whole regex thing is looking for env variable, if the config is e.g. like this:
`download_path=$DIRECTORY`.
But the whole logic for getting the env variables is already in the `shishito_support.get_opt()` method, no?

Reply:
nice I ddint know it is alreay implemented in `shishito_support.get_opt()`
anyway that regex was legacy code